### PR TITLE
{data,lib}[intel/2017a] ROOT v6.10.04, CFITSIO v3.41

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

edit: now requires https://github.com/easybuilders/easybuild-easyblocks/pull/1236 due to ROOT version bump to latest 6.10.04 which requires use of `cmake`...

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2049.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/99fe088a6b9bc72ffef73137011fe999 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2443.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e3f09b43759feb86ff04a2dbb77cc337 for a full test report.

File: easybuild/easyconfigs/c/CFITSIO/CFITSIO-3.41-GCCcore-6.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description = """CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
+FITS (Flexible Image Transport System) data format."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+toolchainopts = {'pic': True}
+
+srcversion = '%s0' % version.replace('.', '')
+source_urls = ['http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/']
+sources = ['%%(namelower)s%s.tar.gz' % srcversion]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/c/CFITSIO/CFITSIO-3.41-GCCcore-6.3.0.eb]: Missing checksum

File: easybuild/easyconfigs/r/ROOT/ROOT-6.06.02-intel-2017a-Python-2.7.13.eb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+toolchainopts = {'pic': True}
+
+sources = ['%(namelower)s_v%(version)s.source.tar.gz']
+source_urls = ['ftp://root.cern.ch/root/']
+patches = [
+    'configure_FftwFromMkl_28.patch',
+    'ROOT-v5_recent-ifort.patch',
+    'ROOT-v5.34.26_libX.patch',
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/r/ROOT/ROOT-6.06.02-intel-2017a-Python-2.7.13.eb]: Missing checksum

File: easybuild/easyconfigs/r/ROOT/ROOT-6.06.02-intel-2017a-Python-2.7.13.eb [hunk 3]
```diff
@@ -0,0 +1,59 @@
+name = 'ROOT'
+version = '6.06.02'
```
Comment (boegel) [inline easybuild/easyconfigs/r/ROOT/ROOT-6.06.02-intel-2017a-Python-2.7.13.eb]: hmm, I should go with the more recent 6.10.04 instead...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2021.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9f99cc547ae27217be3ffb82d6505739 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2474.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b59049dff62c2f77f38849b1347e0b80 for a full test report.

**Pull request merged**